backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/compatibility/AQLInterpreterFactory.java

```diff
@@ -85,11 +85,18 @@ public AQLInterpreter create(ViewExtensionDescription viewExtensionDescription)
         // @formatter:on
 
         for (String qualifiedName : qualifiedNames) {
+            if (qualifiedName.contains("::")) {
+                this.logger.warn("Sirius Web does not support Accelleo-style :: references");
```

Comment:
Typo: "Acceleo" has a single "l"

Reply:
Also, I just noticed the newly added strings are missing the NON-NLS markers. This is normally configured to cause compilation errors. Are you developing inside STS, or some other IDE?

One final note (I think): could you prefix your commit messages with `[cleanup] `?